Tasks/VsTest/taskinputparser.ts

```diff
@@ -179,15 +180,18 @@ function initTestConfigurations(testConfiguration: models.TestConfigurations) {
         }
         if (testConfiguration.vsTestVersion.toLowerCase() === 'toolsinstaller') {
             tl.debug("Trying VsTest installed by tools installer.");
-
+            ci.publishEvent( { subFeature: 'ToolsInstallerSelected', isToolsInstallerPackageLocationSet: !utils.Helper.isNullEmptyOrUndefined(tl.getVariable(constants.VsTestToolsInstaller.PathToVsTestToolVariable)) } );
```

Comment:
don't think we need a separate CI here.. can we fold this into the main CI event ? (you do want to know other versions used as well)

Reply:
Other versions used we can get from our single ci event

Reply:
this was specifically to see if the tasks are being coupled

Reply:
folding this into the single ci events

Reply:
actually no, leaving this here, in the dta flow the single ci will only fire when dta execution host is invoked. That may not happen due to some other error.